process/graduation_criteria.md

```diff
@@ -1,25 +1,25 @@
-== CNCF Graduation Criteria v1.3
+## CNCF Graduation Criteria v1.3
 
 Every CNCF project has an associated maturity level. Proposed CNCF projects should state their preferred maturity level. A two-thirds supermajority is required for a project to be accepted as incubating or graduated. If there is not a supermajority of votes to enter as a graduated project, then any graduated votes are recounted as votes to enter as an incubating project. If there is not a supermajority of votes to enter as an incubating project, then any graduated or incubating votes are recounted as sponsorship to enter as an sandbox project. If there is not enough sponsorship to enter as an sandbox stage project, the project is rejected. This voting process is called fallback voting.
 
-The criteria for each stage of maturity are described below, and there is a separate https://github.com/cncf/toc/blob/main/process/project_proposals.adoc[Project Proposal process].
+The criteria for each stage of maturity are described below, and there is a separate [https://github.com/cncf/toc/blob/main/process/project_proposals.md](Project Proposal process).
```

Comment:
You should use relative links if they're intra-repo: `[/process/project_proposals.md](Project Proposal Process)`

Reply:
I'm going to mark cancel on these because this would then be inconsistent across the whole repo -- when we do another admin pass, I'll add this.